### DISCUSSION TITLE
VCRUNTIME140.dll Error

### DISCUSSION THREAD

**Discussion by NetherFolk** [glzr-io/glazewm#843] [General] [answered]
I couldn't even open the software.

![Screenshot (1)](https://github.com/user-attachments/assets/bad3b024-9305-455d-b191-bda2e564adb8)
This thing came out instead. It seems Zebar works so it's just a GlazeWM issue.

![Screenshot (2)](https://github.com/user-attachments/assets/15b4c7be-d891-4c89-933d-52ef843c9a56)
UPDATE: There seem to be no configurations or anything for glazewm.

What should I do?

**Comment by oligocene** [comment, marked as answer]
You seem to be missing Visual C++ Redist files, search for the missing DLL, should give you some download links at Microsoft.

GlazeWM generates the config.yaml on first startup if it is missing.

**Reply by NetherFolk** [reply]
Thanks, this helped fix the issue!
